### PR TITLE
[Service Connector]remove firewall rule after creation

### PR DESCRIPTION
**Pull request by xfz11**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by xfz11** [comment]
@kairu-ms Could you help merge the PR?

**Comment by yonzhan** [comment]
@kairu-ms for awareness

File: src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_credential_free.py [hunk 1]
```diff
@@ -207,42 +211,54 @@ def set_user_admin(self, user_object_id, **kwargs):
...
-            self.create_aad_user_in_mysql(connection_kwargs, query_list)
-
-        # remove firewall rule
-        if ip_name is not None:
             try:
+                self.create_aad_user_in_mysql(connection_kwargs, query_list)
+            except AzureConnectionError as e:
+                logger.warning(e)
+                # allow public access
+                self.set_target_firewall(True, ip_name, '0.0.0.0', '255.255.255.255')
```
Comment (yungezz) [inline src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_credential_free.py:230]: need user confirmation on this, how to add a confirmation option here?

Comment (xfz11) [reply src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_credential_free.py:247]: Updated

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**